frontend/src/pages/modelServing/screens/projects/DeleteServingRuntimeModal.tsx

```diff
@@ -47,7 +47,7 @@ const DeleteServingRuntimeModal: React.FC<DeleteServingRuntimeModalProps> = ({
             servingRuntime.metadata.namespace,
           );
 
-          Promise.all<ServingRuntimeKind | K8sStatus>([
+          Promise.allSettled<ServingRuntimeKind | K8sStatus>([
```

Comment:
FWIW -- allSettled no longer has a .catch for issues within' the calls. It will only hit the .catch if it hits some syntax error or something that is not "promise related". Failed promises are in the `.then`.